src/gui/widgets/TabWidget.cpp

```diff
@@ -61,21 +66,31 @@ TabWidget::~TabWidget()
 
 
 
-void TabWidget::addTab( QWidget * _w, const QString & _name, int _idx )
+void TabWidget::addTab( QWidget * _w, const QString & _name, const QString & _tooltip, const char *activePixmap, const char *inactivePixmap, int _idx )
```

Comment:
Function parameters are not prefixed with an underscore anymore, that's an old coding convention, so please change `_tooltip` to `tooltip` etc.